src/utils/deployment.js

```diff
@@ -23,7 +23,9 @@ const deployAll = async (web3, privateKey, logger) => {
     },
     validatorSet: {
       owner: deployingAccount.address,
-      initialValidators : [],
+      initialValidators : [
+        '0xc99F38231501Ca5F2e4c31d5b9Df8060B8182Ba5'
+      ],
```

Comment:
?

Reply:
What is src/utils/deployment.js used for?

Reply:
Dev-deploy script uses it. It can be used while creating local network mock to see node running against real network.